answer.txt

```diff
@@ -0,0 +1,4 @@
+• What is a gitignore file and what should it include for a Java project?
```

Comment:
Bro this commit was to be merge to master already. The pr was to just be answers